homa_api.c

```diff
@@ -115,7 +115,7 @@ ssize_t homa_recv(int sockfd, void *buf, size_t length, int flags,
 	args.source_addr = *src_addr;
 	args.flags = flags;
 	args.id = *id;
-	result = ioctl(sockfd, HOMAIOCRECV, &args);
+	result = homa_recvp(sockfd, &args);
```

Comment:
Is there some benefit that accrues from this extra layer of call? It's not a huge overhead, but it's a nonzero overhead.

Reply:
The compiler will inline the function. there is no overhead. The benefit is that when instrumenting the system calls you only have to do it in one place. The subsequent commits for benchmarking are an example of instrumenting the syscalls.

Reply:
see https://godbolt.org/z/1nazGbTfb

Reply:
Interesting. I bow in the face of superior data...